app/src/main/java/com/sun/moviedb_45/ui/home/HomeCategoryAdapter.java

```diff
@@ -0,0 +1,89 @@
+package com.sun.moviedb_45.ui.home;
+
+import android.databinding.DataBindingUtil;
+import android.databinding.ObservableArrayList;
+import android.databinding.ObservableList;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.sun.moviedb_45.R;
+import com.sun.moviedb_45.data.model.Movie;
+import com.sun.moviedb_45.databinding.ItemHomeBinding;
+
+public class HomeCategoryAdapter extends RecyclerView.Adapter<HomeCategoryAdapter.ViewHolder> {
+    private ObservableList<ObservableList<Movie>> mMovies;
+    private ObservableList<String> mCategory;
+    private CategoryListener mListener;
+
+    public HomeCategoryAdapter(CategoryListener listener) {
+        mListener = listener;
+        mMovies = new ObservableArrayList<>();
+        mCategory = new ObservableArrayList<>();
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        LayoutInflater layoutInflater = LayoutInflater.from(viewGroup.getContext());
```

Comment:
```suggestion
        layoutInflater = LayoutInflater.from(viewGroup.getContext());
```
nền pass context vào adapter để không phải tạo layoutInflater nhiều lần